doc/chapters/api/index.rst

```diff
@@ -80,7 +82,7 @@ Launches the specified application/code in the container.
 |
 
 :Return Values:
-        :pid: ``int32`` PID of the process run inside the container.
+        :pid: ``int32`` the external PID of the process run inside the container.
```

Comment:
Alright. I read it and felt it needed something. Your suggestion might be even better. I will change it.